gtkm/stats/get_stats/fetch.py

```diff
@@ -0,0 +1,219 @@
+                user_surname = user_name_data.split()[1]
+        else:
+            user_name = None
+            user_surname = None
+
+        return user_name, user_surname
+
+    # RESPOSITORY BASIC INFO
+
+    async def _get_user_repos_info(self, URL: str = None) -> None:
+        repos_info = await get_endpoint_data(URL)
+        JSON_repos_info = json.loads(repos_info)
+
+        stargaze_count = 0
+        forks_count = 0
+        for element in JSON_repos_info:
+            stargaze_count = stargaze_count + element.get("stargazers_count")
+            forks_count = forks_count + element.get("forks_count")
+
+        self.user_data_json_file['stargaze_count'] = stargaze_count
+        self.user_data_json_file['repos_count'] = len(JSON_repos_info)
+        self.user_data_json_file['forks_count'] = forks_count
+
+
+class GithubFetchRepositoryData(ConfigBase):
+
+    # Fetcher config class
+    CONFIG_DIR = os.path.dirname(__file__) + "/config"
+    PATH = CONFIG_DIR + "/github_config.json"
+
+    URL_BASE = "https://api.github.com"
+
+    repositires_data_json_file: json = {}
+
+    def __init__(self, gtkm_cookie):
+        self.gtkm_cookie = gtkm_cookie
+        super().__init__(self.PATH)
+
+    async def execute_parsing(self):
+        user_name = await self._get_user_name()
+
+        for data_part in self.config["repos info"]:
+            parsing_data = getattr(self, "_get_" + data_part["function"])
+
+            status = await parsing_data(
+                self.URL_BASE + str(data_part["URL"]).format(user_name))
+
+        return self.repositires_data_json_file
+
+    async def _get_repos_data_info(self, URL: str = None) -> None:
+
+        language_list_URL = "/repos/{}/{}/languages"
+        user_name = await self._get_user_name()
+
+        users_repositories = await get_endpoint_data(URL)
+
+        JSON_basic_user_data = json.loads(users_repositories)
+
+        temp_final_json = []
+
+        for repository in JSON_basic_user_data:
+            temp_json: json = {}
+
+            temp_json["repo_language_list"] = json.loads(
+                await
+                get_endpoint_data(self.URL_BASE + language_list_URL.format(
+                    user_name, repository.get("name"))))
+            temp_json["repo_language_list_user"] = json.loads(
+                await
+                get_endpoint_data(self.URL_BASE + language_list_URL.format(
+                    user_name, repository.get("name"))))
+
+            temp_json["repo_owner"] = repository.get("owner")["login"]
+            temp_json["repository_name"] = repository.get("name")
+            temp_json["repo_url"] = repository.get("html_url")
+
+            temp_json["stargaze_count"] = repository.get("stargazers_count")
+            temp_json["forks_count"] = repository.get("forks_count")
+
+            temp_json["watchers_count"] = repository.get("watchers_count")
+
+            temp_json["contributors_count"] = repository.get("watchers_count")
```

Comment:
Te dwie linie (160 i 161) chyba nie są potrzebne, gdy przypisanie `contributors_count` jest w linii 164, a `watchers_count` w 158.

Reply:
Owszem. Powiem więcej, tam są pomylone pola pliku jsonowego i w trzech po kolei jest to samo.